### DISCUSSION TITLE
Exp 01 : Put "Visit a frozen planet" in place of "Visit a green star"  ?

### DISCUSSION THREAD

**Discussion by leodium** [cwmonkey/nms-expeditions#5] [Ideas]
Put "Visit a frozen planet" (Exp02Normal Ph03 Milestone04) in place of "Visit a green star" (Exp01Redux Ph03 Milestone07)?
And fill in the correct reward? ^RS_S2_S3M3 --> ^RS_S1_S3M7

> [
>   {
>     "Amount": 1,
>     "CantClaimRewardDescription": "",
>     "CantRewardMessage": "^INTRCT_NOROOM_L",
>     "Description": "^UI_EXPED_BIOME_COLD_DESC",
>     "DescriptionDone": "^UI_EXPED_BIOME_COLD_DESC_D",
>     "Encryption": {
>       "DecryptMissionId": "^",
>       "Description": "^",
>       "HoverPopupIcon": {
>         "Filename": ""
>       },
>       "IsEncrypted": false,
>       "Patch": {
>         "Filename": ""
>       },
>       "Subtitle": "^",
>       "TitleUpper": "^"
>     },
>     "Icon": {
>       "Filename": "textures/ui/frontend/icons/expedition/patch.biome.3.dds"
>     },
>     "IconGrey": {
>       "Filename": ""
>     },
>     "Id": "^",
>     "IsRendezvous": false,
>     "IsStageControl": false,
>     "MilestoneIndex": 0,
>     "Mission": "^BIOME_COLD",
>     "MissionIcon": {
>       "Filename": "textures/ui/frontend/icons/missions/expeditionmissions/mission.planets.on.dds"
>     },
>     "MissionIconNotSelected": {
>       "Filename": "textures/ui/frontend/icons/missions/expeditionmissions/mission.planets.off.dds"
>     },
>     "MissionIconSelected": {
>       "Filename": "textures/ui/frontend/icons/missions/expeditionmissions/mission.planets.on.dds"
>     },
>     "Reward": "**^RS_S1_S3M7**",
>     "RewardDescription": "",
>     "StageIndex": 0,
>     "Title": "^UI_EXPED_BIOME_COLD_NAME",
>     "TitleUpper": "^UI_EXPED_BIOME_COLD_NAME_U"
>   }
> ]

**Comment by cwmonkey** [comment]
That would still just auto complete since you start on a frozen planet. The idea is to NOT auto complete any missions.

**Comment by leodium** [comment]
Oups ... 
A radioactive planet, then?

> {
>   "Amount": 1,
>   "CantClaimRewardDescription": "",
>   "CantRewardMessage": "^INTRCT_NOROOM_L",
>   "Description": "^UI_EXPED_BIOME_RAD_DESC",
>   "DescriptionDone": "^UI_EXPED_BIOME_RAD_DESC_D",
>   "Encryption": {
>     "DecryptMissionId": "^",
>     "Description": "^",
>     "HoverPopupIcon": {
>       "Filename": ""
>     },
>     "IsEncrypted": false,
>     "Patch": {
>       "Filename": ""
>     },
>     "Subtitle": "^",
>     "TitleUpper": "^"
>   },
>   "Icon": {
>     "Filename": "textures/ui/frontend/icons/expedition/patch.biome.5.dds"
>   },
>   "IconGrey": {
>     "Filename": ""
>   },
>   "Id": "^",
>   "IsRendezvous": false,
>   "IsStageControl": false,
>   "MilestoneIndex": 0,
>   "Mission": "^BIOME_RAD",
>   "MissionIcon": {
>     "Filename": "textures/ui/frontend/icons/missions/expeditionmissions/mission.planets.on.dds"
>   },
>   "MissionIconNotSelected": {
>     "Filename": "textures/ui/frontend/icons/missions/expeditionmissions/mission.planets.off.dds"
>   },
>   "MissionIconSelected": {
>     "Filename": "textures/ui/frontend/icons/missions/expeditionmissions/mission.planets.on.dds"
>   },
>   "Reward": "**^RS_S1_S3M7**",
>   "RewardDescription": "",
>   "StageIndex": 0,
>   "Title": "^UI_EXPED_BIOME_RAD_NAME",
>   "TitleUpper": "^UI_EXPED_BIOME_RAD_NAME_U"
> }

**Reply by cwmonkey** [reply]
Yeah, that case would make more sense, but of the quests that make sense as a replacement, are any of them really "better" than the others? The main problem is you are on a planet with emeril, so the reward is basically meaningless. I'm not terribly worried about which milestone is there, as long as it doesn't auto complete. Otherwise I could go back and forth all day testing a "best" when really there is none objectively.

What I will do, however, is see if there is a way to shift the expedition to a yellow star by messing with the hashes, then I can simply restore the proper milestone.

**Comment by leodium** [comment]
Ok ... This is beyond my capabilities ... I understand the copy/paste idea (copy elsewhere/replace here) but for the rest of the scripts and interactions ... I've still got a way to go ...

Otherwise, there's also a MileStone "Visit a pirate system" in expedition 06 Ph02 MileStone03 =)

I'll take a look at it too ... Just to be curious and test too =)

**Reply by cwmonkey** [reply]
It is currently beyond me, too. I'm just thinking I *might* be able to do it if I go to a nearby yellow star and if I can find some hashes that look similar to the ones in the json in my save file. I figure I have a 20% chance of it actually working out that way.

But like I said, the reward is a bunch of emeril, so there isn't much point in over-thinking which milestone to use, since they are all basically pointless unless we can change the starter planet.

**Reply by leodium** [reply]
Personally, if HG spun emeril ... Maybe it's just to compensate for the fact that the engine that makes it possible to reach the green system is made from Emeril?

Sometimes, rewards are used to solve a subsequent MileStone ... We shouldn't overthink things either (^^)

**Reply by cwmonkey** [reply]
Starting on a green star is a bug, you're supposed to start on a yellow star, so yes reaching a green star should give you the mats to get to the blue star. Starting on a green star also messes up the flow of the whole expedition. The idea is you go yellow, red, green, blue (last rendezvous), so fixing that is the real priority.

**Comment by leodium** [comment]
Isn't it with this line that the reward is established? 

You can put whatever you like, can't you? As long as the value exists in the game database?!?

ex : "Reward": "^RS_S1_S3M7",

**Reply by cwmonkey** [reply]
Yep, switching the reward is a good idea. I'll go dig something up later.

**Reply by leodium** [reply]
I'm already going to finish listing the stages and rewards of the different missions in my gDrive Sheet. And then, parallel the codes/names/values for both. With this inventory, I could piece together a new adventure like a puzzle and you could find alternatives more easily

**Comment by cwmonkey** [comment]
Thankfully, this conversation is now moot: 8a981332e13f5cfa73bd39d9fcbcc0a51ae6ece3
